clang/test/Sema/warn-thread-safety-analysis.c

```diff
@@ -29,6 +29,13 @@ struct LOCKABLE Mutex {};
 
 struct Foo {
   struct Mutex *mu_;
+  int  a_value GUARDED_BY(mu_);
```

Comment:
@pdherbemont I think you should be checking **both** late and regular parsing for all attributes that you're adding support for.

Reply:
done!